### DISCUSSION TITLE
Have a button in the body of text to toggle light/dark mode?

### DISCUSSION THREAD

**Discussion by ArcTrooper210** [squidfunk/mkdocs-material#7755] [Q&A]
Is there a way to add an icon that is within the text body that functions the same as the light/dark mode toggle in the header?

**Comment by kamilkrzyskow** [comment]
Hello @ArcTrooper210,
this should be possible using extra JavaScript that would be executed on button press inside your content. To make it sync with the button in the header, I would make it so that the logic of your new button simply clicked on the palette button, instead of hooking into / creating custom logic for the palette change.

References which could help you:
- https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript
- https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#examples
- https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors
- https://github.com/squidfunk/mkdocs-material/blob/60c0dc1b8af71cef004eb7a91ca6ea8bae1e76cc/src/templates/partials/palette.html#L24
